pyproject.toml

```diff
@@ -29,7 +29,8 @@ requires-python = '>=3.10'
 dependencies = [
    'imagecodecs',
    'numpy',
-   'tifffile>=2020.5.7',
+   'tifffile>=2023.9.26',
```

Comment:
Is there a specific feature/requirement causing the bump in version number?

Reply:
Yes, loading (and lazy writing) levels / zarr / dask